_posts/2022-03-16-launch-announcement-1-3-0.markdown

```diff
@@ -0,0 +1,109 @@
+
+One of the major new additions to 1.3.0 is a set of tools for observability between logs, metrics, and other live data. When critical, event-based information is correlated with other application or system data and reported centrally, availability issues are easier to identify and resolve. Try these new tools out (see Observability, below) and let the community know what you think.
+
+Here is a list of major and minor feature enhancements that have been included in this release:
+
+### Index State Management
+
+* **Continuous Mode in Transform**: With this release, you can now run transforms in a continuous mode based on a schedule. This enables you to keep an incremental transform job running on a group of source indices actively ingesting new data. At each run, only the modified buckets will be transformed, reducing the overhead for users who want to their transform up to date.
+
+### Observability
+
+Customers can now create custom Observability Applications to view the availability status of all their systems, where log events can be combined with trace and metric data, into a single view of system health empowering developers and IT Ops to resolve issues faster and with fewer escalations.
+
+* **App Analytics**: In the past, users had to collate logs, traces, and metrics in separate views which made application monitoring difficult. With App Analytics Dashboards, users can now view application logs, traces, and metrics in one view instead of moving between different visualizations.
+
+![App Analytics Dashboard]({{ site.baseurl }}/assets/media/blog-images/2022-03-16-Launch-Announcement-1-3-0/App-Analytics.gif){: .img-fluid }
+
+* **Trace ID Correlation**: Users who regularly track events across applications will now be able to use correlation ID (based on the [Open Telemetry specification](https://opentelemetry.io/docs/reference/specification/logs/overview/#log-correlation)) to tie events together when viewing events and in-context visualizations.
+
+![Trace log Analytics]({{ site.baseurl }}/assets/media/blog-images/2022-03-16-Launch-Announcement-1-3-0/trace-log-correlation.gif){: .img-fluid }
+
+* **Live Tail**: Previously, when users were watching a live event take place, they had to manually refresh their view. Users can now configure the interval in which content is refreshed saving the hassle of manually refreshing.
+
+![Live Tail View]({{ site.baseurl }}/assets/media/blog-images/2022-03-16-Launch-Announcement-1-3-0/Live-Tail.gif){: .img-fluid }
+
+* **Field Insights and Enhanced Visualization Support**: Visualization types such as Pie and heat maps were added as well as the ability to add named threshold markers as lines on the visualizations. When users want to know more about a metric that they are looking at, they can select view surrounding events to get a correlated picture. In the event a user needs a reference point for a future meeting, visualizations can now be saved to notebooks for convenience.
```

Comment:
"they can select view surrounding events to get a correlated picture" -> Something doesn't parse for me. Is "view surrounding events" a button or something?

Reply:
"View surrounding events" is a link that users can select from.  The UX should be clearer.

<img width="793" alt="image" src="https://user-images.githubusercontent.com/98365234/158729204-3fc759b5-e1c8-4af5-a4a9-f54cfd79c58b.png">

Visualization types such as pie and heat maps were added as well as the ability to add named threshold markers as lines on the visualizations. When users want to know more about a metric that they are looking at, they can select "View surrounding events" to get a correlated picture.  In the event a user needs a reference point for a future meeting, visualizations can now be saved to notebooks for convenience.

Reply:
that looks good @brijos